lib/mongoid/extensions/range.rb

```diff
@@ -78,7 +77,7 @@ def __mongoize_hash__(object)
           hash.slice!('min', 'max', 'exclude_end')
           hash.compact!
           hash.transform_values!(&:mongoize)
-          hash
+          hash.blank? ? nil : hash
```

Comment:
This line should be reverted. What if I intentionally want to insert an empty hash to Mongoid?